admin/services/data-access/check.data.service.js

```diff
@@ -277,4 +277,42 @@ checkDataService.sqlFindAllFormsUsedByPupils = async function (pupilIds) {
   return byPupil
 }
 
+/**
+ * Find all the started checks that have not been processed
+ * @returns {Boolean}
+ */
+checkDataService.sqlHasUnprocessedStartedChecks = async function () {
```

Comment:
@GuyHarwood @jon-shipley shall I move these two methods under psychometrician data service?
I think it makes sense based on our conversation earlier

Reply:
providing they are not used elsewhere, yes

Reply:
Yes please